allennlp/commands/predict.py

```diff
@@ -94,62 +102,95 @@ def _get_predictor(args: argparse.Namespace) -> Predictor:
 
     return Predictor.from_archive(archive, args.predictor)
 
-def _run(predictor: Predictor,
-         input_file: IO,
-         output_file: Optional[IO],
-         batch_size: int,
-         print_to_console: bool) -> None:
 
-    def _run_predictor(batch_data):
+class _Predict:
```

Comment:
I don't find this name terribly descriptive, `_PredictorWrapper` maybe? Something that gives more of a sense of what this class is for